OWNERS

```diff
@@ -9,6 +9,8 @@ filters:
     - mfojtik
     - marun
     - tnozicka
+    - mrunalp
+    - sjenning
 
     # approvers are limited to the team that manages rebases and pays the price for carries that are introduced
```

Comment:
@rphillips This comment suggest against addition as approvers.

Reply:
These individuals have had approval rights within Origin to perform approvals on this section of code before. Not allowing Seth or Mrunal approval rights here will greatly slow down the team, since it will be dependent on non-owners of the kubelet code to approve and review changes.

Was there a discussion on slack or the dev list about approval rights here?

Reply:
> Not allowing Seth or Mrunal approval rights here will greatly slow down the team, since it will be dependent on non-owners of the kubelet code to approve and review changes.

The solution is to build kubelet out of another repo/branch. It is was networking team and oc team do. As long as o/k is the build source, we have this rule and are very hesitant to weaken it. We understand the pain though, totally.

Reply:
> Not allowing Seth or Mrunal approval rights here will greatly slow down the team

Where has this actually happened? 

I would claim that approvers here do a very good job to react quickly to approval requests. If approval is rejected, there should be good reasons. I would personally be curious where you had the impression in the past where we did not live up to the standard.